from queue_logic.exceptions import MyError


class ViberError(MyError):
    ...


class UsesError(MyError):
    ...
